### DISCUSSION TITLE
Add ServiceStack Reference generated Swift DTOs not binding due to letter case mismatch.

### DISCUSSION THREAD

**Discussion by Eljay8878** [ServiceStack/Discuss#90] [Q&A]
The generated Typed Swift DTOs contains camelCase property/ data members and doesn't contain Codingkeys enums to handle this. The ServiceStack API DTOs contains PascalCase members. 

When consuming the service it doesn't decode and bind the members. Only when I manually add the coding key enums. Is there some setting/configuration option within e.g. within the SwiftGenerator class to alter the generation behavior for this?

**Comment by mythz** [comment]
If you have an active commercial license you can report issues to https://github.com/ServiceStack/Issues
Please include minimal code (e.g. C# DTOs and Swift DTOs) to repro any issues when submitting a report.
